styles.css

```diff
@@ -0,0 +1,5 @@
+h1 {
+
+    font-family: sans-serif;
+    font-size: 12em;
```

Comment:
12 em is echt te klein, dit moet minimaal 16 em zijn. 
Misschien leuk om er een rood kleurtje bij te doen  👍🏻 😄